cypress/support/Actions.js

```diff
@@ -27,6 +26,13 @@ export function CreateNewAction({ response, expectedEntities, requiredEntities,
   if (requiredEntities) actionModal.TypeRequiredEntities(requiredEntities)
   if (disqualifyingEntities) actionModal.TypeDisqualifyingEntities(disqualifyingEntities)
   actionModal.ClickCreateButton()
+}
+
+export function CreateNewActionThenVerifyInGrid({ response, expectedEntities, requiredEntities, disqualifyingEntities, uncheckWaitForResponse, type = 'TEXT' }) {
```

Comment:
Maybe not required for this review but is related to the idea that the functions/helpers should be further decoupled. In other words, you should be able to control when you create/add an action independently of when you verify it's representation in the list.

I think previously `CreateNewAction` was implicitly doing both things.  Now you made it so you can call `CreateNewAction` without verifying.  Likewise, you would also want to be able to verify an action in the grid with being forced to create one.

Example:
```
const newAction = { ... }
actions.CreateNewAction(newAction )
actions.VerifyInGrid(newAction)
```
Still ensures same is created and verified.

This is an example of something that would be more in line with our plan to migrate the tests to a more flexible feature based organization.  If we change how actions are displayed and modify the list or remove it but don't touch anything related to creating actions the function for creating actions should still work.

Reply:
Lets discuss this point via email...and later.